packages/instructor-stream/src/utils/tokenizer.ts

```
@@ -152,490 +150,487 @@ export default class Tokenizer {
+    } else if (typeof input === 'string') {
+      buffer = this.encoder.encode(input)
+    } else if ((typeof input === 'object' && 'buffer' in input) || Array.isArray(input)) {
+      buffer = Uint8Array.from(input)
+    } else {
+      /** Signal error and exit early instead of throwing locally */
+      return this.error(
+        new TypeError(
           'Unexpected type. The `write` function only accepts Arrays, TypedArrays and Strings.'
         )
-      }
+      )
```

Biome (2.1.2)

[error] The function should not return a value because its return type is void.

The function is here:

'void' signals the absence of value. The returned value is likely to be ignored by the caller.

(lint/correctness/noVoidTypeReturn)